config/samples/sriovnetwork_v1_ovsnetwork.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
please make a valid example here

Reply:
Hey, yury on PTO - ill address it in followup

Reply:
since yury is on PTO i addressed this in #696

Reply:
please note that other samples have the same value. i have aligned this one in the followup pr